koku/masu/external/downloader/azure/azure_report_downloader.py

```diff
@@ -57,17 +57,15 @@ def get_initial_dataframe_with_date(
         tracing_id (str): The tracing id
     """
     dh = DateHelper()
-    try:
-        data_frame = pd.read_csv(local_file, usecols=["UsageDateTime"])
-        time_interval = "UsageDateTime"
-        date_format = "%Y-%m-%d %H:%M:%S"
-        base_cols = utils.INGRESS_REQUIRED_COLUMNS
-    except ValueError:
+    date_format = "%Y-%m-%d %H:%M:%S"
```

Comment:
Right that is true, but we need the format captured for when we convert it to a date later, to prevent the old `unconverted data remains` here `if datetime.datetime.strptime(interval, date_format) >= process_date:`